src/web/streaming/components/stream/stream.tsx

```diff
@@ -17,15 +17,18 @@ export const Stream = ({ id, user }: { id: string; user: ChatUserType }) => {
   }
 
   return (
-    <Flex>
-      <Box flexGrow={1}>
+    <Flex minH={'100vh'}>
+      <Box>
```

Comment:
The same staff, if you need to reuse elsewhere you, you will be blocked by those styles.